src/cryptmd5.ts

```diff
@@ -66,6 +66,17 @@ export class Hash {
     constructor(public value: string) { }
 }
 
+
+export function janus_hash(password: string, salt: string): Hash {
+    if('' === password) {
+        return new Hash('');
+    } else {
+        return crypt_md5(password, salt);
+    }
+}
+
+
+
 /**
```

Comment:
Hmm, not sure if this belongs to cryptmd5.ts module. This has nothing to do with neither crypt(3) nor md5.

I would rather suggest something like this at the call-site:
```ts
let plaintext: string = '';
let hash: Hash = plaintext.length > 0 ? crypt_md5(plaintext, 'salt') : new Hash('');
```

What do you think?